Project.toml

```
@@ -15,7 +15,7 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 CSV = "0.5, 0.6, 0.7, 0.8"
 CodecZlib = "0.4, 0.5, 0.6, 0.7"
 DataFrames = "0.15, 0.16, 0.17, 0.18, 0.19, 0.20, 0.21, 0.22"
-FileIO = "1"
+FileIO = "< 1.6"
```

Regardless of the problems with an upper bound mentioned above, probably versions < 1 should be excluded here.

---

Ah, yeah I get your point above, but still seems better than not doing anything, no?
And I'm clearly missing something; why < 1?

---

Since they are currently not compatible, they should not be compatible with future releases either.

---

Hmm, but won't what I've done also ensure that?

---

No, it means `[0.0.0, 1.6.0)`: https://julialang.github.io/Pkg.jl/v1/compatibility/#Inequality-specifiers

---

But that does ensure that it's not compatible with future releases?

---

Don't you mean releases >= 1.6?

---

I'm confused now :smile: I just mean that the bound "< 1.6" is not correct since it means "[0.0.0, 1.6.0)" whereas you actually want "[1.0.0, 1.6.0)".

---

Not completely sure if it is valid, but it would correspond to "[1.0.0, 1.6.0)":
```suggestion
FileIO = "~1.0, ~1.1, ~1.2, ~1.3, ~1.4, ~1.5"
```

---

Ah, _now_ I follow!